cypress.json

```diff
@@ -1,6 +0,0 @@
-{
-  "pluginsFile": "tests/e2e/plugins/index.js",
-  "baseUrl": "http://localhost:8080",
-  "testFiles": ["out_of_game/**/*.js", "in_game/**/*.js"],
-  "video": false
-}
```

Comment:
We'll need to be smarter about telling cypress where to load the config file from in its github action. From the looks of the logs, CI failed because cypress failed to find the config file when looking for cypress.json